pkg/model/api/v1/view.go

```diff
@@ -0,0 +1,84 @@
+// Copyright 2024 The Perses Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package v1
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"gopkg.in/yaml.v2"
+)
+
+var _ = json.Unmarshaler(&View{})
+var _ = yaml.Unmarshaler(&View{})
```

Comment:
we never needed to do that for other struct / resource definition. Why did you need it ?

Reply:
It's not _strictly_ necessary. Just a habit to show the intention that `View` implements those interfaces. I can remove it if you want

Reply:
oh I didn't know this trick. Interesting !
But yeah can you remove it as we don't do it across the whole code. (just to be consistent).